configs/index.json

```diff
@@ -0,0 +1,57 @@
+{
+  "remote_base_path": "https://raw.githubusercontent.com/d-roak/madara/feat/configs-index/configs/",
+  "genesis_assets": [
+    {
+      "name": "Account.json",
+      "md5": "bbdfc1b69f76e9c2784db6d8b697dc56"
```

Comment:
hash of the contents of the whole file. Easier to check on CI

Reply:
But two different files (indentation, spacing, order of the keys) can lead to two different md5 hashes, while still representing the same contract.
Won't cause a problem?

What exactly are we checking in the CI?

Reply:
That the files that we are retrieving in the URL match the md5 that is in the `index.json` (simple security validation). Since you can define custom url's for each file

Reply:
Answering the first question, that's no problem. We want to verify the integrity of the file, not if it's contract X or Y